socorro/processor/mozilla_transform_rules.py

```diff
@@ -1022,7 +1022,7 @@ def _get_pretty_os_version(self, processed_crash):
         major_version = int(version_split[0])
         minor_version = int(version_split[1])
 
-        if processed_crash.os_name == 'Windows':
+        if processed_crash.os_name.lower().startswith('windows'):
```

Comment:
Hmm... Sorry but this makes me nervous all of a sudden. 
Suddenly it HAS to ALWAYS be a string. Otherwise it will fail on that `.lower()`. 
Before, if the crash some crazy and screwed up (which isn't unrealistic) there was a chance that the value of `processed_crash.os_name` might be `None` (or perhaps some other crap like a number). Then this processor rule would fail here. 
The test above (on line 1007) only checks if the key is present. It doesn't check that its non-falsy. 

Sorry, but I don't feel confident around this. It feels like it's a lot more important that processor rules don't raise exceptions. 

Please have a think about it and if you don't know with confidence either perhaps it should be:
```python
if (
    processed_crash.os_name and 
    processed_crash.os_name.lower().startswith('windows')
):
```